features/polyExplorer/src/context/explorer-context.jsx

```diff
@@ -190,31 +190,26 @@ export const ExplorerProvider = ({ children }) => {
     }
 
     function handleOnboardingPopupClose() {
-        changeNavigationState({ firstRun: false });
+        setPopUp(null);
         writeFirstRun(false);
     }
 
     function handleOnboardingPopupMoreInfo() {
-        handleOnboardingPopupClose();
-        history.push("/info");
+        createPopUp({ type: "info-main" });
+        writeFirstRun(false);
```

Comment:
OK, it makes sense that  if it's onboarding you don't add anything to history, right?

Reply:
No it should be a popup now and not a "screen" in history